test/common/testlib.py

```diff
@@ -2070,11 +2070,11 @@ def restore_file(self, path: str, post_restore_action: Optional[str] = None):
 
         if self.file_exists(path):
             backup = os.path.join(self.vm_tmpdir, path.replace('/', '_'))
-            self.machine.execute("mkdir -p %(vm_tmpdir)s; cp -a %(path)s %(backup)s" % {
-                "vm_tmpdir": self.vm_tmpdir, "path": path, "backup": backup})
-            self.addCleanup(self.machine.execute, "mv %(backup)s %(path)s" % {"path": path, "backup": backup})
+            self.machine.execute(f"mkdir -p {self.vm_tmpdir}; cp -a {path} {backup}")
+            # self.addCleanup(self.machine.execute, f"rm -f {path}s; mv {backup} {path}")
```

Comment:
Let's not commit commented out code (unless it is part of some larger comment or documentation).

Reply:
Oops, that was a leftover. Fixed.